fastlane/lib/fastlane/swift_fastlane_api_generator.rb

```diff
@@ -177,7 +185,7 @@ def generate_lanefile_parsing_functions
   } else {
       potentialDictionary = fromString
   }
-  let dictionary: [String : String] = try! JSONSerialization.jsonObject(with: potentialDictionary.data(using: .utf8)!, options: []) as! [String : String]
+  let dictionary: [String : Any] = try! JSONSerialization.jsonObject(with: potentialDictionary.data(using: .utf8)!, options: []) as! [String : Any]
```

Comment:
this is scary, but I think you're right. It will return a Hash that can be a hash of a hash of a ha....